src/library/scala/collection/mutable/OpenHashMap.scala

```diff
@@ -81,6 +81,9 @@ extends AbstractMap[Key, Value]
     h ^ (h >>> 7) ^ (h >>> 4)
   }
 
+  /** Increase the size of the table.
+    * Copy only the occupied slots, effectively eliminating the deleted slots.
+    */
```

Comment:
> in case you make changes to this PR

It's closed already. (But thanks for joining us!)

> Minor nitpick

That's what Scala IDE wants to do to my code, too, but I'm just following [the style guide](http://docs.scala-lang.org/style/scaladoc.html). And [the other style guide](https://wiki.scala-lang.org/display/SW/Syntax). Can't have too much style.